src/createRollupConfig.ts

```diff
@@ -74,9 +88,7 @@ export function createRollupConfig(
     // Establish Rollup output
     output: {
       // Set filenames of the consumer's package
-      file: `${paths.appDist}/${safePackageName(
-        opts.name
-      )}.${format}.${env}.js`,
+      file: outputName,
       // Pass through the file format
       format,
```

Comment:
You may add the same condition here to highlight migration to `esm`

Reply:
Well, I am not sure to call it a migration. Or at least babel plugin typings expect the "es" only. Strangely enough, the rollup expects "esm" in the format option. I wonder how is it possible it actually worked till now. It's a bit of mess.

https://rollupjs.org/guide/en#command-line-flags

Reply:
Babel do not use `es` value anywhere. The type is defined in this module.

Reply:
Format is being passed to Babel in here ... https://github.com/palmerhq/tsdx/blob/943648edb06cd131f2c1751120cd1301804b480b/src/createRollupConfig.ts#L163

Either way, I've used the same condition for a rollup format.

Reply:
I mean babelOptions is defined here too. Format is only used to check non-commonjs
https://github.com/palmerhq/tsdx/blob/d111c131a5e5cf19d62b98e123060c2ab55c43da/src/createRollupConfig.ts#L16-L50

Reply:
Doh! How could I've missed that :) Ok, it's now consolidated as "esm" everywhere with a check for "es" to avoid breakage.